1. Unit testing/eslint.config.js

```diff
@@ -0,0 +1,56 @@
+import js from "@eslint/js";
+import globals from "globals";
+import tseslint from "typescript-eslint";
+import { defineConfig } from "eslint/config";
+
+export default defineConfig(
+  [
+    {
+      files: [
+        "**/*.{js,mjs,cjs,ts,mts,cts}",
+      ],
+      plugins: {
+        js,
+      },
+      extends: [
+        "js/recommended",
+      ],
+    },
+    {
+      files: [
+        "**/*.{js,mjs,cjs,ts,mts,cts}",
+      ],
+      languageOptions:
+        {
+          globals:
+            globals.browser,
+        },
+    },
+    tseslint.configs
+      .recommended,
+    {
+      files: [
+        "**/*.{ts,tsx,cts,mts}",
+      ],
+      rules: {
+        "@typescript-eslint/no-explicit-any":
+          "off",
+        "@typescript-eslint/no-unused-vars":
+          "off", //default annoying errors, not the focus of this task
+        "no-console":
+          "warn",
+        "curly": [
+          "error",
+          "all",
+        ],
```

Comment:
bardziej ciekawostka, ale takie rzeczy raczej ustawia się prettierem i wyłącza się je w eslincie - jak dwie rzeczy odpowiadają za formatowanie to robi się bałagan.

https://prettier.io/docs/integrating-with-linters